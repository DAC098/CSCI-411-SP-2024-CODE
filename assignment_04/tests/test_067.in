5
2 3 2
aaaataaaatacgaaaa accgggcctctg
aaaaaaaaaaatgg aaaa
ccgtccgttgc tcg
qxxddwqllxmohqmkqgpdb nyyxgxyvjcswrczujpxhwdkhhdnudaux
ccacccattccctcaccac atgagacttaac
