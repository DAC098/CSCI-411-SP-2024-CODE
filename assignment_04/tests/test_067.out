aaaataaaatacgaaaa
_accgggcct_c___tg
33
aaaaaaaaaaatgg
_______aaaa___
30
ccgtccgttgc
___t_c___g_
24
__qx_x___ddw_qllxmohqmk__qgpd__b
nyyxgxyvjcswrczujpxhwdkhhdnudaux
52
ccacccattccctcaccac
__a_tga__gactta__ac
31
